src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -417,9 +417,8 @@ private static bool RemovalMayIntroduceInterpolationAmbiguity(ParenthesizedExpre
                         node.Expression.Kind() == parentBinaryExpression.Kind() &&
                         parentBinaryExpression.Right == node)
                     {
-                        return !node.IsSafeToChangeAssociativity(
-                            node.Expression, parentBinaryExpression.Left,
-                            parentBinaryExpression.Right, semanticModel);
+                        return !CSharpSemanticFacts.Instance.IsSafeToChangeAssociativity(
+                            (BinaryExpressionSyntax)node.Expression, parentBinaryExpression, semanticModel);
```

Comment:
this cast is safe as we checked that we have a binary-expr of the smae kind in the if-check.